gradle.properties

```diff
@@ -23,9 +23,9 @@ gsonVersion=2.8.8
 ballerinaGradlePluginVersion=0.14.1
 
 ballerinaLangVersion=2201.0.1
-stdlibTaskVersion=2.2.1
-stdlibLogVersion=2.2.1
-stdlibIoVersion=1.2.1
-stdlibTimeVersion=2.2.1
-observeVersion=1.0.2
-observeInternalVersion=1.0.2
+stdlibTaskVersion=2.2.2-20220510-134400-67a7a7f
+stdlibLogVersion=2.3.0-20220510-164700-f61e84f
```

Comment:
Since we are updating the log version to a minor, we need to update the module version also to a minor one.

Reply:
Updated